packages/utilities/src/index.ts

```diff
@@ -39,6 +39,7 @@ export * from './lib/range';
 export * from './lib/regExpEsc';
 export * from './lib/roundNumber';
 export * from './lib/splitText';
+export { AbortError, sleep, sleepSync, type SleepOptions } from './lib/sleep';
```

Comment:
Now that `DOMException` is gone:
```suggestion
export * from './lib/sleep';
```